e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -978,7 +978,7 @@ describeEE("scenarios > embedding > dashboard appearance", () => {
 
     getIframeBody().findByText("Rows 1-21 of first 2000").should("exist");
 
-    cy.get("#iframe").then($iframe => {
+    cy.get("#iframe").should($iframe => {
```

Comment:
Using `should` for assertion is ideal in this case. [See the doc](https://docs.cypress.io/api/commands/should#Whats-the-difference-between-then-and-shouldand).